docs/tutorial.md

```diff
@@ -0,0 +1 @@
+# DerivativeFreeSolvers.jl Tutorial
```

Comment:
Better to remove this file.

Reply:
Why?

Reply:
It's an empty tutorial. If it's just barebones docs, we don't need this file.

Reply:
It’s there to remind us to write one :-)